src/components/ProgramRecord/RecordsHelp.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from '@edx/frontend-platform/i18n';
+import { Hyperlink } from '@edx/paragon';
+
+const RecordsHelp = ({ helpUrl }) => (
+  <section className="help">
+    <h2>
+      <FormattedMessage
+        id="help.section.header"
+        defaultMessage="Questions about Learner Records?"
+        description="A header for the help section"
+      />
+    </h2>
+    <p>
+      <FormattedMessage
+        id="help.section.content.with.link"
+        defaultMessage="To learn more about records you can "
+        description="Content for the help section with a link to the records help area"
+      />
+      <Hyperlink
+        destination={helpUrl}
+        target="_blank"
+        showLaunchIcon={false}
+      >
+        <FormattedMessage
+          id="link.to.records.help.area"
+          defaultMessage="read more in our records help area."
+          description="A link to the records help area"
+        />
+      </Hyperlink>
+    </p>
```

Comment:
This is another are I'm concerned about translations. Having this message split into multiple chunks could make it a nightmare to translate in certain languages. 

I think we want to see if there is another approach we can take here.